src/widgets/ProfileWidget/subcomponents/ConnectedProfileWidget/ConnectedProfileWidget.tsx

```diff
@@ -0,0 +1,149 @@
+  const { collectionToken, collectionImage, collectionName } = useAppSelector((state) => state.config);
+  const { avatarUrl } = useAppSelector((state) => state.user);
+  const { tokens } = useAppSelector((state) => state.balances);
+  const { isLoading, ownedTokenMeta } = useAppSelector((state) => state.profile);
+
+  const [ownedNfts, setOwnedNfts] = useState<CollectionTokenInfo[]>([]);
+  const [searchValue, setSearchValue] = useState('');
+
+  // If there is a user active, then we need to check the id param to see if it's the same as the acccount
+  const { active, account, deactivate } = useWeb3React<Web3Provider>();
+  // If there is no id param, then we should display the current user's profile
+  const { id } = useParams();
+  const isPrivateProfile = !id || id === account;
+
+  const dispatch = useAppDispatch();
+  const ensAddress = useEnsAddress(account || '');
+
+  useEffect(() => {
+    // console.log('ProfileWidget - balances: ', balances);
+    // console.log('ProfileWidget - tokens: ', tokens);
+    if (tokens) {
+      dispatch(fetchOwnedTokenMeta({ library, collectionToken, tokenIds: tokens }));
+    }
+  }, [library, collectionToken, tokens]);
+
+  useEffect(() => {
+    if (ownedTokenMeta) {
+      console.log('ProfileWidget - ownedTokenMeta: ', ownedTokenMeta);
+      setOwnedNfts(ownedTokenMeta);
+    }
+  }, [ownedTokenMeta, isLoading]);
+
+  const handleDisconnectClick = () => {
+    deactivate();
+  };
+
+  if (false) {
+    console.log('ProfileWidget - Wallet connected: ', active ? 'Yes' : 'No');
+    console.log('ProfileWidget - isPrivate Profile: ', isPrivateProfile ? 'Yes' : 'No');
+    // console.log('ProfileWidget - ownedNfts: ', ownedNfts);
+  }
+
+  return (
+    <div className={`profile-widget ${className}`}>
+      <ProfileHeader
+        avatarUrl={avatarUrl}
+        backgroundImage={collectionImage}
+        ensAddress={ensAddress}
+        address={account || ''}
+        onLogoutButtonClick={handleDisconnectClick}
+      />
+      <div className="profile-widget__button-group-container">
+        <div className="profile-widget__button-group">
+          <Button text="NFTs" className="profile-widget__button-group__button profile-widget__button-group__button--is-active" />
+          <Button text="ACTIVITY" className="profile-widget__button-group__button" disabled />
+          <Button text="LISTED" className="profile-widget__button-group__button" disabled />
+        </div>
+      </div>
+      <div className="profile-widget__content">
+        <div className="profile-widget__search-bar-container">
+          <i className="profile-widget__search-bar-icon">
+            <IconSearch />
+          </i>
+          <Input className="profile-widget__search-bar" placeholder="Search NFT" onChange={e => setSearchValue(e.target.value)} />
+        </div>
+        <div className="profile-widget__collections">
+          <Accordion
```

Comment:
> You don't have to use the `Accordion` component since we won't have multiple collections **for now**.

Future proofing.. and it matches the design.

Reply:
It just doesn't make sense to be able to hide and expand the single collection we have now.

Reply:
:heavy_check_mark: